.github/workflows/CI.yml

```diff
@@ -169,7 +169,7 @@ jobs:
         cxx: [icpc]
     env:
       MACOS_HPCKIT_URL: >-
-        https://registrationcenter-download.intel.com/akdlm/irc_nas/18242/m_HPCKit_p_2021.4.0.3389_offline.dmg
+        https:/registrationcenter-download.intel.com/akdlm/IRC_NAS/a99cb1c5-5af6-4824-9811-ae172d24e594/m_HPCKit_p_2023.1.0.44543.dmg
```

Comment:
Should it be:
```suggestion
        https://registrationcenter-download.intel.com/akdlm/IRC_NAS/a99cb1c5-5af6-4824-9811-ae172d24e594/m_HPCKit_p_2023.1.0.44543.dmg
```

Reply:
Sorry I hadn't noticed that! I did a copy-and-paste from the @fortran-lang setup-fortran repository: 

https://github.com/fortran-lang/setup-fortran/blob/2a11c57ecf26d40ea43d66a33f9f6fc4d593d6f8/setup-fortran.sh#L402

Reply:
I will open a PR on that too, although it works, it looks like a typo